src/pages/documents/document-slot.vue

```diff
@@ -45,4 +45,5 @@
   display flex
   flex 1
   flex-direction column
+  overflow hidden
```

Comment:
Is this repairing something? It also seems to have side effects on `zi-popover`.